examples/complete/variables.tf

```diff
@@ -28,12 +28,6 @@ variable "resource_group" {
   default     = null
 }
 
-variable "resource_tags" {
-  type        = list(string)
-  description = "Optional list of tags to be added to any created resources"
-  default     = []
-}
```

Comment:
why did you remove this? Please add it back and use it with the secrets manager module